mitmproxy/tools/console/master.py

```diff
@@ -123,12 +123,30 @@ def get_editor(self) -> str:
         else:
             return "vi"
 
+    def get_hex_editor(self) -> str:
+        if m := os.environ.get("MITMPROXY_EDITOR"):
+            return m
+        if m := os.environ.get("EDITOR"):
+            return m
```

Comment:
Do you think it makes sense to use `$EDITOR` here?

Reply:
I assumed that the user may have a preferred utility to edit binary formats.

Reply:
`$EDITOR` is set on most Linux systems, if we always take that as the first choice we would essentially never open a real hex editor. I don't think we should take it into account. We could check for a `$MITMPROXY_HEXEDITOR` variable though.